### PR TITLE
Clean up solr when destroying an exhibit.

### LINKED ISSUE #202: Deleting a collection doesn't remove the solr records associated with it.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #202.

### CODE REVIEW

File: app/controllers/exhibits_controller.rb [hunk 1]
```diff
@@ -7,6 +7,11 @@ def ingest_members
     ExhibitProxy.new(@exhibit).reindex
   end
 
+  def destroy
+    @exhibit.resources.destroy_all
```
Comment (cbeer) [inline app/controllers/exhibits_controller.rb]: I wonder if we should add this to the `Spotlight::Exhibit` model?

Comment (tpendragon) [inline app/controllers/exhibits_controller.rb]: I assumed you didn't have it there for some use case - is that not true?

Comment (cbeer) [inline app/controllers/exhibits_controller.rb]: Off-hand, I can't think of a reason to leave an orphaned database row. Cleaning up Solr, however, is probably best left to the downstream model, like you've done here.

Comment (tpendragon) [inline app/controllers/exhibits_controller.rb]: I'd have to do some testing to make sure dependent: :destroy_all will do what I want - the exhibit needs to exist in order for the solr IDs to be generated.

**Pull request merged**